src/libs/OptionsListUtils.ts

```diff
@@ -1689,6 +1689,17 @@ function getValidPersonalDetailOptions(
     return personalDetailsOptions;
 }
 
+function getRestrictedLogins(config: GetOptionsConfig, options: OptionList, canShowManagerMcTest: boolean): Record<string, boolean> {
+    const userHasReportWithManagerMcTest = Object.values(options.reports).some((report) => isManagerMcTestReport(report));
+    return {
+        [CONST.EMAIL.MANAGER_MCTEST]:
+            !canShowManagerMcTest ||
+            (getIsUserSubmittedExpenseOrScannedReceipt() && !userHasReportWithManagerMcTest) ||
+            !Permissions.canUseManagerMcTest(config.betas) ||
+            isUserInvitedToWorkspace(),
+    };
+}
+
```

Comment:
Im wondering about other logins, like concierge or notifications, etc. that are also omitted